src/types/message-types.ts

```diff
@@ -75,6 +76,13 @@ export type QueryResultEntry = GenericMessage & {
   encodedData?: string;
 };
 
+export type GenericMessageSubscriptionHandler = (message: GenericMessage) => void;
```

Comment:
The prefix seems to not add much value, consider:
```suggestion
export type SubscriptionHandler = (message: GenericMessage) => void;
```
or
```suggestion
export type MessageSubscriptionHandler = (message: GenericMessage) => void;
```